### DISCUSSION TITLE
Pixel Art

### DISCUSSION THREAD

**Discussion by AVolkoff6** [bnpr/Malt#177] [Q&A]
How would you achieve a pixel art effect similar to the one in your video?

**Comment by pragma37** [comment]
I guess you mean this video?
https://youtu.be/k_8e_j_B8mM

For that scene I used a custom render pipeline.
https://pragma37.gumroad.com/l/malt-gallery-3

Right now, the easiest way to do pixel art is to simply set the *Viewport > Resolution Scale* at a value lower than 1 and disable *Viewport > Smooth Interpolation*.
![imagen](https://user-images.githubusercontent.com/36610999/143255066-947a1a3e-b31f-40ef-99ff-b07039dab3c9.png)
You may also want to set the *Samples > Grid Size* to 1, or lower the *Samples > Width*.
